airbase/pom.xml

```diff
@@ -969,6 +969,7 @@
                         <gitDescribe>
                             <tags>true</tags>
                         </gitDescribe>
+                        <injectAllReactorProjects>true</injectAllReactorProjects>
```

Comment:
thank you!
can you please move this to a separate PR that I can approve?
And add CHANGES entry.

Reply:
https://github.com/airlift/airbase/pull/412